src/app/legacy/psammead/psammead-navigation/src/ScrollableNavigation/index.jsx

```diff
@@ -6,7 +6,7 @@ import {
   GEL_GROUP_2_SCREEN_WIDTH_MIN,
   GEL_GROUP_2_SCREEN_WIDTH_MAX,
 } from '#psammead/gel-foundations/src/breakpoints';
-import { C_WHITE } from '#psammead/psammead-styles/src/colours';
+import { C_WHITE, C_BLACK } from '#psammead/psammead-styles/src/colours';
```

Comment:
Can we import `BLACK` from https://github.com/bbc/simorgh/blob/latest/src/app/components/ThemeProvider/palette.ts#L2